### PR TITLE
adding whisper readme changes for v1

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
/cc @MichaelClifford

### CODE REVIEW

File: recipes/audio/audio_to_text/README.md [hunk 1]
```diff
@@ -1,23 +1,26 @@
 # Audio to Text Application
 
-  This sample application is a simple recipe to transcribe an audio file.
-  This provides a simple recipe to help developers start building out their own custom LLM enabled
-  audio-to-text applications. It consists of two main components; the Model Service and the AI Application.
+
+  This recipe helps developers start building their own custom LLM enabled audio transcription applciations. It consists of two main components: the Model Service and the AI Application.
```
Comment (MichaelClifford) [inline recipes/audio/audio_to_text/README.md]:
```suggestion
  This recipe helps developers start building their own custom AI enabled audio transcription applications. It consists of two main components: the Model Service and the AI Application.
```
I'm not sure its technically an LLM.

File: recipes/audio/audio_to_text/README.md [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
 
   There are a few options today for local Model Serving, but this recipe will use [`whisper-cpp`](https://github.com/ggerganov/whisper.cpp.git)
   and its included Model Service. There is a Containerfile provided that can be used to build this Model Service within the repo,
-  [`model_servers/whispercpp/Containerfile`](/model_servers/whispercpp/Containerfile).
+  [`model_servers/whispercpp/Containerfile`](/model_servers/whispercpp/base/Containerfile).
+
+ The AI Application will connect to the Model Service via its OpenAI compatible API. The recipe relies on [Langchain's](https://python.langchain.com/docs/get_started/introduction) python package to simplify communication with the Model Service and uses [Streamlit](https://streamlit.io/) for the UI layer. You can find an example of the chat application below.
```
Comment (MichaelClifford) [inline recipes/audio/audio_to_text/README.md]:
```suggestion
 The AI Application will connect to the Model Service via an API. The recipe relies on [Langchain's](https://python.langchain.com/docs/get_started/introduction) python package to simplify communication with the Model Service and uses [Streamlit](https://streamlit.io/) for the UI layer. You can find an example of the chat application below.
```

File: recipes/audio/audio_to_text/README.md [hunk 3]
```diff
@@ -26,7 +26,36 @@ In order to build this application we will need a model, a Model Service and an
 * [Interact with the AI Application](#interact-with-the-ai-application)
     * [Input audio files](#input-audio-files)
 
-### Download a model
+## Quickstart
+
+To run the application with pre-built images from `quay.io/ai-lab`, use `make quadlet`. This command
+builds the application's metadata and generates Kubernetes YAML at `./build/chatbot.yaml` to spin up a Pod that can then be launched locally.
```
Comment (MichaelClifford) [inline recipes/audio/audio_to_text/README.md:32]:
```suggestion
builds the application's metadata and generates Kubernetes YAML at `./build/audio-to-text.yaml` to spin up a Pod that can then be launched locally.
```

File: recipes/audio/audio_to_text/README.md [hunk 4]
```diff
@@ -1,22 +1,21 @@
...
-  Our AI Application will connect to our Model Service via it's API endpoint.
+The AI Application will connect to the Model Service via an API. The recipe relies on [Langchain's](https://python.langchain.com/docs/get_started/introduction) python package to simplify communication with the Model Service and uses [Streamlit](https://streamlit.io/) for the UI layer. You can find an example of the chat application below.
 
-<p align="center">
-<img src="../../../assets/whisper.png" width="70%">
-</p>
+
+![](/assets/whisper.png) 
+
+## Try the Chat Application: Coming soon
```
Comment (MichaelClifford) [inline recipes/audio/audio_to_text/README.md]:
```suggestion
## Try the Audio to Text Application:
```

File: recipes/audio/audio_to_text/README.md [hunk 5]
```diff
@@ -1,22 +1,21 @@
...
 
-  There are a few options today for local Model Serving, but this recipe will use [`whisper-cpp`](https://github.com/ggerganov/whisper.cpp.git)
-  and its included Model Service. There is a Containerfile provided that can be used to build this Model Service within the repo,
-  [`model_servers/whispercpp/Containerfile`](/model_servers/whispercpp/Containerfile).
+There are a few options today for local Model Serving, but this recipe will use [`whisper-cpp`](https://github.com/ggerganov/whisper.cpp.git) and its included Model Service. There is a Containerfile provided that can be used to build this Model Service within the repo, [`model_servers/whispercpp/base/Containerfile`](/model_servers/whispercpp/base/Containerfile).
 
-  Our AI Application will connect to our Model Service via it's API endpoint.
+The AI Application will connect to the Model Service via an API. The recipe relies on [Langchain's](https://python.langchain.com/docs/get_started/introduction) python package to simplify communication with the Model Service and uses [Streamlit](https://streamlit.io/) for the UI layer. You can find an example of the chat application below.
```
Comment (MichaelClifford) [inline recipes/audio/audio_to_text/README.md]:
```suggestion
The AI Application will connect to the Model Service via an API. The recipe relies on [Langchain's](https://python.langchain.com/docs/get_started/introduction) python package to simplify communication with the Model Service and uses [Streamlit](https://streamlit.io/) for the UI layer. You can find an example of the audio to text application below.
```

File: recipes/audio/audio_to_text/README.md [hunk 6]
```diff
@@ -103,39 +99,37 @@ To convert your input audio files to 16-bit WAV format you can use `ffmpeg` like
 ffmpeg -i <input.mp3> -ar 16000 -ac 1 -c:a pcm_s16le <output.wav>
 ```
 
-### Embed the AI Application in a Bootable Container Image
+## Embed the AI Application in a Bootable Container Image
 
-To build a bootable container image that includes this sample chatbot workload as a service that starts when a system is booted, cd into this folder
-and run:
+To build a bootable container image that includes this sample chatbot workload as a service that starts when a system is booted, run: `make -f Makefile bootc`. You can optionally override the default image / tag you want to give the make command by specifying it as follows: `make -f Makefile BOOTC_IMAGE=<your_bootc_image> bootc`.
```
Comment (MichaelClifford) [inline recipes/audio/audio_to_text/README.md]:
```suggestion
To build a bootable container image that includes this sample audio to text workload as a service that starts when a system is booted, run: `make -f Makefile bootc`. You can optionally override the default image / tag you want to give the make command by specifying it as follows: `make -f Makefile BOOTC_IMAGE=<your_bootc_image> bootc`.
```

File: recipes/audio/audio_to_text/README.md (hunk 4 above)
Comment (Gregory-Pereira) [reply recipes/audio/audio_to_text/README.md]: good catch

File: recipes/audio/audio_to_text/README.md (hunk 6 above)
Comment (Gregory-Pereira) [reply recipes/audio/audio_to_text/README.md]: As per slack comment no makefile = no bootc. Dropped from readme

**Review by MichaelClifford — APPROVED** [review]
LGTM

**Pull request merged**